docs/DeveloperGuide.md

```diff
@@ -567,10 +568,11 @@ _Similar to UC10 except without extension 2b._
 1.  The system should work on any _mainstream OS_ as long as it has Java `17` or above installed.
 2.  The system should be able to hold up to 1000 contacts without a noticeable sluggishness in performance for typical usage.
 3.  The system should be developed in a modular way for easier updates and bug fixes.
-4.  The system should ensure data consistency accross all instances.
+4.  The system should ensure data consistency across all instances.
 5.  The system should continue functioning in the event of a missing or corrupted save file.
 6.  The system should encrypt sensitive data to follow data protection laws.
 7.  The interface should be intuitive and easy to use.
+8.  The system should allow the user to easily populate their previous command.
```

Comment:
I believe this should not be a non-functional requirement but rather as a use case since it is a function